doc/release-notes/dash/release-notes-0.12.2.2.md

```diff
@@ -173,7 +173,7 @@ See detailed [change log](https://github.com/raptoreum/raptoreum/compare/v0.12.2
 ### Network/Sync:
 - [`5d58dd90c`](https://github.com/raptoreum/raptoreum/commit/5d58dd90c) Make sure to clear setAskFor in Raptoreum submodules (#1730)
 - [`328009749`](https://github.com/raptoreum/raptoreum/commit/328009749) fine-tune sync conditions in getblocktemplate (#1739)
-- [`362becbcc`](https://github.com/raptoreum/raptoreum/commit/362becbcc) Bump MIN_PEER_PROTO_VERSION to 70208 (#1772)
+- [`362becbcc`](https://github.com/raptoreum/raptoreum/commit/362becbcc) Bump OLD_MIN_PEER_PROTO_VERSION to 70208 (#1772)
```

Comment:
Did you mean to update the old Dash release notes?

Reply:
no but we need to make our own release note which we have not done